### ISSUE TITLE
Using variables within a http client receive

### ISSUE THREAD

**Issue by lebuitrago** [citrusframework/citrus#676]
**Citrus Version**
2.8.0

**Expected behavior**
Specifying a variable name within an http.receive().validate() should extract the value of the variable and use that as the final value. 

For example:

```
variable("VERSION_PATH", "//root/version/@number");

http(http -> http
                .client(SomeServiceEndpoint)
                .receive()
                .response(HttpStatus.OK)
                .validate("${VERSION_PATH}", "1.0")

```
The ${VERSION_PATH} in the .validate should be replaced by its value = '//root/version/@number'.

**Actual behavior**
The .validate step is completely bypassed on execution and as a result, a failure is never caught, even if the expected result is wrong.

In other words, the use of a variable in the spot above is not working (or supported?).

However, if I specified the following, it would successfully validate:

```
http(http -> http
                .client(SomeServiceEndpoint)
                .receive()
                .response(HttpStatus.OK)
                .validate("//root/version/@number", "1.0")

```

**Test case sample**
> Please see above for my sample.

Is this expected behavior, perhaps not supported?

Thank you.

**Comment by vijayakumarj** [comment]
@christophd - We at our framework as well facing the same issue. Please resolve it.
